### PR TITLE
Feat: Grid system On Live

### PR DESCRIPTION
**Pull request by Rishi05051997**
Peer members please review code

### CODE REVIEW

**Review by SHIKHASINGH1506 — APPROVED** [review]
Everything on place. Good to merge

File: styles/simplified-grid.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  grid-row: row;
+ }
+  
+
+  
+    
+
+  
+
+
```
Comment (Mohasin-Haque) [inline styles/simplified-grid.css]: Remove whitespaces..

**Review by Mohasin-Haque — APPROVED** [review]
Everything seems good to me.. You can go ahead but needs to go through comments once.

File: styles/simplified-grid.css (hunk 1 above)
Comment (Rishi05051997) [reply styles/simplified-grid.css]: Ok Removing now

**Pull request merged**